### PR TITLE
Make tests headless with the help of xvfb

### PR DESCRIPTION
**Pull request by Hainish**
I'm not sure if this is categorically useful.  There may be instances where you actually want to have the ability to interact with the webdriver browser.  We could also add this as an option, like `--headless` if desired.

### CODE REVIEW

**Comment by semenko** [comment]
Hey cool, thanks! This definitely could be useful, especially as we're trying to expand testing.

I'll need to think about this a bit more (and not sure what's up with Travis failing)

**Comment by Hainish** [comment]
Since the xvfb daemon was run as a travis `before_install` step, the addition of xvfb-run was causing the tests to fail.  Looks like it's working now..

**Comment by semenko** [comment]
Cool! I'm thinking adding xvfb-run into the actual test scripts is going to break testing on Mac -- thoughts?

**Comment by Hainish** [comment]
This latest commit should fix that.  It won't be headless for OSX users, though.

File: test/chromium.sh [hunk 1]
```diff
@@ -11,6 +11,10 @@ else
     git rev-parse && cd "$(git rev-parse --show-toplevel)"
 fi
 
+# Make sure we have xvfb-run and it's not already set.
+if [ ! -v XVFB_RUN -a -n "$(which xvfb-run)" ]; then
```
Comment (semenko) [inline test/chromium.sh]: Errors for me:
```
++ which xvfb-run
+ '[' '!' -v XVFB_RUN -a -n '' ']'
./chromium.sh: line 15: [: -v: unary operator expected
```

**Comment by Hainish** [comment]
@semenko try now?

**Comment by semenko** [comment]
Awesome! :fireworks:

**Pull request merged**